docs/libvirt-pod-networking.md

```diff
@@ -0,0 +1,131 @@
+# Using CNI Interfaces and libvirt for networking
+
+Author: Fabian Deutsch \<fabiand@redhat.com\>
+
+## Introduction
+
+Today networking in KubeVirt is not integrated with Kubernetes. In order to
+have network connectivity inside the VM, a user must create a specific VM Spec
+that induces a particular libvirt domxml, in order to reuse a pre-exising host
+network interface.
+
+This is obviously suboptimal:
+
+- It is not integrated with Kubernetes
+- It does not provide the VM with connectivity to pods
+- It requires to specify host specific bits (the network device to use)
+
+The purpose of this document is to describe one approach of how we can provide
+a minimal integrated solution, with few workarounds.
+
+**Note:** This approach is rough, and not optimized, but the rule of thumb was
+to find a robust solution which works with Kubernetes today.
+
+
+## Use-case
+
+The primary use-case is to allow VM to get connected to the pod network.
+
+
+## API
+
+The API is pretty clear: As usual an `interface` element will be defined.
+To indicate that pod interfaces shall be used for networking, a new network
+type (`pod-network`) is introduced.
```

Comment:
how shall we specify _which_ of the possible pod networks is to be connected to this `interface`? (once there are multiple ones)